virttest/virsh.py

```diff
@@ -642,15 +642,15 @@ def __str__(self):
 class EventTracker(object):
 
     @staticmethod
-    def start_get_event(vm_name):
+    def start_get_event(vm_name, event_cmd='event %s --all --loop'):
         """
         Use a virsh session with subcommand 'event' to catch events
-
         :param vm_name: name of the vm to be catched
+        :param event_cmd: cmd to check event
         :return: the virsh session with 'event'
         """
         virsh_session = aexpect.ShellSession(VIRSH_EXEC)
-        event_cmd = 'event %s --all --loop' % vm_name
+        event_cmd = event_cmd % vm_name
```

Comment:
I wonder if event_cmd need vm_name every time or not... Should it be event_cmd = event_cmd.format(vm_name)? What do you think?

Reply:
Hi yingshun ,Thanks for your patient. Could u please help to tell what's the difference `between "format" and "%s"` here. I find that if we use empty vm_name for "%s" type , we can also get ` event  --all --loop` @Yingshun

Reply:
start_get_event() will be used by wait_event() and some virsh utilities will use this decorator, so I don't think it's reasonable to pass an empty vm_name here. The only difference between 'format' and '%' here is that there will no exception if no need to pass vm_name to 'event_cmd'.
For eg:
```
>>> event_cmd = "event --all --loop"
>>> my_vm = 'test1'
>>> event_cmd % my_vm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> event_cmd.format(my_vm)
'event --all --loop'
>>> 
```

Reply:
updated yingshun,Thanks for your review. Did I get your point?